readme.md

```diff
@@ -43,4 +43,13 @@ Example:
             // ...
         }
     }
+```
+
+# Settings
+
+In some cases, you may need to disable events handling. To disable event handling dynamically set property IsHandlingDisabled from Forte.EpiEasyEvents.Configuration namespace to true.
```

Comment:
Could you update this block and add info about DisabledEventsHandlerScope usage?